packages/gasket-plugin-otel/README.md

```diff
@@ -0,0 +1,49 @@
+# @gasket/plugin-otel
+
+Gasket plugin for OpenTelemetry support. The instrumentation file supports automatic instrumentation for Node.js applications. For more information, see the OTel [Automatic Instrumentation][automatic-instrumentation] documentation. To fine tune traces a manual instrumentation is required.
+
+## Installation
+
+#### Existing apps
+
+```shell
+npm i @gasket/plugin-otel
+```
+
+Copy the [instrumentation](./generator/instrumentation.js) file content to a new file root of your application.
+
+Configure OTel environment variables in your `.env` file:
+
+```shell
+OTEL_EXPORTER_OTLP_ENDPOINT=<your-ess-otel-endpoint>
+OTEL_EXPORTER_OTLP_HEADERS="Authorization=Bearer <ess-secret>"
+OTEL_SERVICE_NAME=<service-name>
+
+# Optional
+OTEL_SERVICE_VERSION=<service-version>
+OTEL_NODE_ENABLED_INSTRUMENTATIONS=<instrumentations>
+OTEL_NODE_RESOURCE_DETECTORS=<resource-detectors>
```

Comment:
this is really exciting. Where would be find out ess otel endpoint? Referencing those docs here would be helped.

Reply:
Another thing to note, these values are used during deploys from ASM:

```diff
- --build-arg ELASTIC_APM_SERVER_URL=${{ steps.get-secrets.outputs.essp_apm_deployment_apm_server_url }} \
- --build-arg ELASTIC_APM_SECRET_TOKEN=${{ steps.get-secrets.outputs.essp_apm_deployment_apm_secret_token }} \

+ --build-arg OTEL_EXPORTER_OTLP_ENDPOINT=${{ steps.get-secrets.outputs.essp_apm_deployment_apm_server_url }} \
+ --build-arg OTEL_EXPORTER_OTLP_HEADERS="Authorization=Bearer ${{ steps.get-secrets.outputs.essp_apm_deployment_apm_secret_token }}" \
```

Reply:
would the endpoint end in `https://<your-ess-instance>/api/apm/otel`? when you note that it is `your-ess-otel-endpoint` I thought there was a specific route for the endpoint.